Library/Formula/yank.rb

```diff
@@ -0,0 +1,18 @@
+class Yank < Formula
+  desc "Yank terminal output to clipboard"
+  homepage "https://github.com/mptre/yank"
+  url "https://github.com/mptre/yank/archive/v0.4.0.tar.gz"
+  # version "0.4.0"
+  sha256 "1adf1b07f7cb9401daeed7a05bad492db8ed77ead4728d9b45f541d56bc2e8c5"
+
+  def install
+    ENV["PREFIX"] = prefix
+    ENV["YANKCMD"] = "pbcopy"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/yank", "-v"
+    system "man", "yank"
```

Comment:
@mikemcquaid @bfontaine a really poor attempt at making a test to test functionality by going over all environmental keys and seeing if <Control-N> then <Enter> will output the correct value. (<Control-N> is \016 in octal apparently) It doesn't work as of now because somehow the `#!/usr/bin/expect -f` line is simply ignored by Ruby. https://gist.github.com/dongcarl/136f19f7c4ecc348d87c